rules/windows/defense_evasion_untrusted_driver_loaded.toml

```diff
@@ -16,14 +29,73 @@ index = ["logs-endpoint.events.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Untrusted Driver Loaded"
+note = """## Triage and analysis
+
+### Investigating Untrusted Driver Loaded
+
+Windows Driver Signature Enforcement (DSE) is a security feature introduced by Microsoft to enforce that only signed drivers can be loaded and executed into the kernel (ring 0). This feature was introduced to prevent attackers from loading their malicious drivers on targets. If the driver has an invalid signature, the system will not allow it to be loaded.
+
+This protection is essential for maintaining the security of the system. However, attackers or even administrators can disable this feature and load untrusted drivers, as this can put the system at risk. Therefore, it is important to keep this feature enabled and only load drivers from trusted sources to ensure the integrity and security of the system.
+
+This rule identifies an attempt to load an untrusted driver, which effectively means that the Driver Signature Enforcement was disabled or bypassed. This can indicate that the system was compromised.
```

Comment:
```suggestion
This rule identifies an attempt to load an untrusted driver, which effectively means that DSE was disabled or bypassed. This can indicate that the system was compromised.
```